crates/rspack_core/src/chunk_group.rs

```diff
@@ -175,6 +175,48 @@ impl ChunkGroup {
     false
   }
 
+  pub fn replace_chunk(&mut self, old_chunk: &ChunkUkey, new_chunk: &ChunkUkey) -> bool {
+    if let Some(runtime_chunk) = self.runtime_chunk {
+      if runtime_chunk == *old_chunk {
+        self.runtime_chunk = Some(new_chunk.clone());
+      }
+    }
+
+    if let Some(entry_point_chunk) = self.entry_point_chunk {
+      if entry_point_chunk == *old_chunk {
+        self.entry_point_chunk = Some(new_chunk.clone());
+      }
+    }
+
+    match self.chunks.iter().position(|x| x == old_chunk) {
+      // when old_chunk doesn't exist
+      None => false,
+      // when old_chunk exists
+      Some(old_idx) => {
+        match self.chunks.iter().position(|x| x == new_chunk) {
+          // when new_chunk doesn't exist
+          None => {
+            self.chunks[old_idx] = new_chunk.clone();
```

Comment:
These clone is unnecessary, you could run `cargo lint` to get more advice by `cargo clippy` , which is required by our ci checking.

Reply:
cool!